src/Analyser/ResultCache/ResultCacheManager.php

```diff
@@ -601,4 +606,8 @@ private function getStubFiles(): array
 		return $stubFiles;
 	}
 
+	private function buildCacheFilePath(): string
+	{
+		return $this->resultCachePath . '/resultCache' . $this->getMetaAsString() . 'php';
```

Comment:
This is the relevant change.
The metadata is encoded as string and included in the cache filepath so result-caches for multiple metadata configurations can co-exist.

Reply:
```suggestion
		return $this->resultCachePath . '/resultCache' . $this->getMetaAsString() . '.php';
```
A small little fix ;)